### PR TITLE
Add support for overriding the OS_REGION_NAME parameter

### PR DESCRIPTION
**Pull request by cmondo**
The Subcloud Name Reconfiguration feature includes a mechanism to override the OS_REGION_VALUE on the dcmanager side. This creates confusion when configuring the region value in the deployment file.

This commits adds the changes to override the OS_REGION_NAME parameter but on the DM side.

Test Plan:

PASS - Deploy a new Subcloud and check DM configuration to
       validate endpoint access. There should be no endpoint
       errors in DM logs and the subcloud should be operational.
PASS - Deploy DM to central and verify that the region has not
       been replaced, since the condition applies only to the
       subcloud role

### CODE REVIEW

File: docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml
Comment (yjian118) [inline docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml:61]: Please use ansible builtin regex module than the sed

File: docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml
Comment (cmondo) [reply docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml]: Done

File: docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml
Comment (cmondo) [reply docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml]: Done

File: docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml
Comment (yjian118) [inline docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml:370]: Will be safer to replace the whole line with "line:"

File: docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml
Comment (yjian118) [inline docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml:369]: The OS_REGION_NAME is kind of static, no need to have the task in line357 and print it out here

**Review by enzocandotti — APPROVED** [review]
LGTM

**Review by hugonicodemos — APPROVED** [review]

**Comment by hugonicodemos** [comment]
I'll wait for @yjian118 approval before merging.

**Review by yjian118 — APPROVED** [review]

**Pull request merged**
